packages/lib/src/plugin_loader.ts

```diff
@@ -26,6 +26,15 @@ export async function loadPluginInfos(pluginList: Map<string, string>) {
 		let pluginInfo: libPlugin.PluginNodeEnvInfo;
 		let pluginPackage: { name?: string, version: string, main?: string, private?: boolean };
 
+		// Check if plugin path exists, otherwise remove it
+		try {
+			require.resolve(pluginPath);
+		} catch (err) {
+			logger.warn(`Plugin path ${pluginPath} does not exist, not loading ${pluginName}`);
```

Comment:
Should we ignore or throw on missing plugin?

Reply:
[As discussed](https://discord.com/channels/450361298220351489/450361298220351491/1310899816502394932), we have agreed to the change in behaviour for missing plugins to be logged and removed rather than throw an error because it allows hosts to remotly (re)start more reliability. However, personally I think this needs to be `logger.error` not `logger.warn` because it indicates that something broke but was recoverable.